Sources/Apodini/WebService.swift

```diff
@@ -78,6 +78,18 @@ extension WebService {
         )
     }
     
+    /// Runs the `SyntaxTreeVisitor` without starting the web service.
+    public func runSyntaxTreeVisitor() {
+        let app = Application()
+        app.logger.info("Running SynatxTreeVisitor to retrieve web service structure.")
+        app.logger.logLevel = .error
+        self.configuration.configure(app)
+        self.register(
+            SemanticModelBuilder(app)
+        )
+        app.logger.logLevel = .debug
```

Comment:
I assume the idea was to restore the previous logLevel state, right?
In that case we would need to properly capture the previous logLevel and not just blindly assume `.debug`.

Reply:
But why change it in the first place? Was the goal to "don't care" about the logs if we don't startup the webservice and just require the structure?
As of now, we specify .debug as the log level for the application logger (non-configurable). Therefore I don't see a huge problem in assuming .debug, but of course the cleaner and more future-proof solution would be to save the previous level and then restore it :)

Reply:
> But why change it in the first place?

I think, it's a matter of silencing otherwise confusing logs? But not sure. Might be confusing if you try to deploy the web service and see log information which implies a started web service, but actually doesn't start the web service(?).

> As of now, we specify .debug as the log level for the application logger (non-configurable).

I think its only `.debug` if compiled in debug configuration? In release configuration it is a less chatty logger, if I remember correctly and it wasn't changed 🙃

Reply:
Yep i thought so too. And in the "old" implementation we didn't care about the, as you put it, chatty logger, since this was executed in its own process?

You're right, i forgot about that. Then the storing of the old level is definitely required.

Reply:
>  And in the "old" implementation we didn't care about the, as you put it, chatty logger, since this was executed in its own process?

Actually I have no clue. But thinking about it it might be that it was indeed not silenced. @lukaskollmer ?
Maybe we just wait for @hendesi to get the rationale behind that 😅

Reply:
The idea was to minimise the output during the synatax tree visit. Since this is just to retrieve the web service structure and not actually starting the web service, i wanted it to be as quite as possible 😅 
But I don't have a super strong opinion about this, so i can also just revert this. If we keep it, then you're right @Supereg  , it should be reverted to the correct level

Reply:
I just reverted the changes made to the log level. There is still an `info` msg that indicates the reason for the output, so this should be fine, I guess